Tests/TuistGeneratorTests/ProjectMappers/ResourcesProjectMapperTests.swift

```diff
@@ -163,7 +163,7 @@ final class ResourcesProjectMapperTests: TuistUnitTestCase {
             .appending(component: Constants.DerivedDirectory.sources)
             .appending(component: "TuistBundle+\(target.name).swift")
         let expectedContents = ResourcesProjectMapper
-            .fileContent(targetName: target.name, bundleName: "\(target.name)Resources", target: target)
+            .fileContent(targetName: target.name, bundleName: "", target: target)
```

Comment:
why empty? 🤔

Reply:
The test shall pass with any value, so this value is irrelevant to this test.

Maybe you have a better idea, or we can keep the original value?

Reply:
Aren't we checking the file content 2 lines below? how is it not relevant? 🤔

Reply:
The `bundleName` is only used `if !target.supportsResources`, in this test, the target is a `framework`, which supports Resources.

Reply:
Here is the code of `ResourcesProjectMapper.fileContent`
```swift
    // swiftlint:disable:next function_body_length
    static func fileContent(targetName: String, bundleName: String, target: Target) -> String {
        if !target.supportsResources {
            return """
            // Bundle name is used here
            // ...

            """
        } else {
            return """
            // Bundle name is not used here.
            // ...
            """
        }
    }

```

Reply:
So if you use `.framework` instead of `staticFramework` for a dependency (you can map it in Dependencies.swift, or Tuist could chose to use it according to the mapping logic), then the bundle name won't match the SPM one?

cc @fortmarek

Reply:
> then the bundle name won't match the SPM one?

If it's framework, then we don't need this bundle name.
Check this [comment](https://github.com/tuist/tuist/blob/685840668d0a63b2e9d22fda753c9c0e1abd85c1/Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift#L144).

> /// Since \(targetName) is a \(target.product), the bundle for classes within this module can be used directly.

Reply:
Hello @fortmarek @kwridan , can you take a look ?

Reply:
Yeah, I believe we don't create a bundle if it's a `framework`, isn't that correct?